### PR TITLE
update explorer

### PR DESCRIPTION
**Pull request by foesa-yang**

### CODE REVIEW

**Review by hetao92 — APPROVED** [review]

File: docs-2.0/nebula-explorer/deploy-connect/ex-ug-connect.md [hunk 1]
```diff
@@ -33,7 +33,7 @@
      - 如果已启用身份验证，但是未创建账号信息，用户只能以 GOD 角色登录，必须填写 `root` 及对应的密码 `nebula`。
```
[the remark this answers is not in the document]
Comment (foesa-yang) [reply docs-2.0/nebula-explorer/deploy-connect/ex-ug-connect.md:33]: If this pr is approved, I will merge this pr to master and 261.

File: docs-2.0/nebula-explorer/deploy-connect/ex-ug-deploy.md [hunk 2]
```diff
@@ -11,17 +22,15 @@
...
-  | 8070 | Nebula-http-gateway 的 HTTP 服务 |
-  
-- 使用的 Linux 发行版为 CentOS ，安装有版本为 v10.16.0 + 以上的 Node.js，安装有版本为1.13及以上的Go。
 
   !!! caution
 
-        目前 Nebula Explorer 提供的包仅在 Linux 环境中使用，如果用户使用 mac 或其他环境，需要克隆 [http-gateway repo](https://github.com/vesoft-inc/nebula-http-gateway)，并修改 `nebula-http-gateway/conf/app.conf` 文件中的 `httpport = 8070` 后使用 `make` 命令编译启动。
+       Explorer 默认使用的端口号为7002，用户可以在安装目录下的 `conf/app.conf` 文件中修改 `httpport`，并重启服务。
+
+- 使用的 Linux 发行版为 CentOS ，安装有版本为1.13及以上的Go。
```
[the remark this answers is not in the document]
Comment (foesa-yang) [reply docs-2.0/nebula-explorer/deploy-connect/ex-ug-deploy.md:33]: centos版本要求我去核对一下

[the remark this answers is not in the document]
Comment (foesa-yang) [reply docs-2.0/nebula-explorer/deploy-connect/ex-ug-deploy.md:33]: 这个他们没有测过，暂时就这么写着吧？

File: docs-2.0/nebula-explorer/deploy-connect/ex-ug-deploy.md [hunk 3]
```diff
@@ -95,37 +122,35 @@ $ sudo rpm -e nebula-graph-explorer-<version>.x86_64
...
-2. 部署 nebula-graph-explorer
+  !!! enterpriseonly
 
-   ```bash
-   $ cd nebula-graph-explorer
-   $ npm run start
-   ```
+        License 仅在企业版提供，请发送邮件至inquiry@vesoft.com。
+
+4. 启动explorer。
```
Comment (abby-cyber) [inline docs-2.0/nebula-explorer/deploy-connect/ex-ug-deploy.md]: before we run the following path, we need to enter the `nebula-explorer` directory, that is run `cd nebula-explorer` before the following command

File: docs-2.0/nebula-explorer/deploy-connect/ex-ug-deploy.md [hunk 4]
```diff
@@ -95,37 +122,35 @@ $ sudo rpm -e nebula-graph-explorer-<version>.x86_64
...
+
+  ```bash
+  $ ./nebula-httpd &
+  ```
+  当屏幕返回以下信息时，表示 Explroer 已经成功启动。
+  ```bash
+  [1] 5169
```
Comment (abby-cyber) [inline docs-2.0/nebula-explorer/deploy-connect/ex-ug-deploy.md:158]: `[1] 5169` is not a fixed output, is it better to add `for example` before the output?

**Review by abby-cyber — APPROVED** [review]

**Review by abby-cyber — APPROVED** [review]

**Review by cooper-lzy — APPROVED** [review]

**Pull request merged**